### DISCUSSION TITLE
Screen lock no work properly

### DISCUSSION THREAD

**Discussion by galliroberto** [MarJose123/filament-lockscreen#69] [Ideas]
In version 2.2 I was able to lock screen by visit /screen/lock
Why I did it? Because I use JS to reset timer.
Now it's impossibile to do it.

---

I see now that I have to call in post /lock-session
Maybe you can change it in GET request

**Comment by MarJose123** [comment]
You need to call it via post method before redirecting to the lock screen page. 

I make this changes to make sure that once a separate tab is unlocked the main tab will not be locked again after it was refreshed.

**Comment by galliroberto** [comment]
Yeah, I saw.
What is the session in the form?

What is the form session?
Is it generated by livewire?

If you want to improve it, you can add support for multi tenant.

**Reply by MarJose123** [reply]
Hi, I will have it here in the discussion since this is not more on issue.

The session was generated by the package itself for its locking mechanism.
